components/atom/radioButton/src/index.scss

```diff
@@ -0,0 +1,64 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$w-atom-radiobutton: 15px !default;
```

Comment:
all hardcoded values should be tokens

Reply:
All values replaced by the proper token but this one:
$bw-atom-radiobutton-checked: 6px !default;

As there is no token w/ this value and I need exactly this value to emulate the effect reflected in the specification